tests/h/traversal/group_test.py

```diff
@@ -17,22 +15,6 @@ def test_it_with_a_group(self, factories, ACL):
         assert context.__acl__() == ACL.for_group.return_value
         ACL.for_group.assert_called_once_with(group)
 
-    @pytest.mark.parametrize(
-        "principal,has_upsert", ((role.User, True), ("other", False))
-    )
-    def test_it_without_a_group(
-        self, set_permissions, pyramid_request, principal, has_upsert
-    ):
-        set_permissions("acct:adminuser@foo", principals=[principal])
-
-        context = GroupContext(group=None)
-
-        assert context.group is None
-        assert (
-            bool(pyramid_request.has_permission(Permission.Group.UPSERT, context))
-            == has_upsert
-        )
-
```

Comment:
I noticed that the role was being used here where it really shouldn't be. There was some old left over tests from before the ACL move.